src/routes/application/application.css

```diff
@@ -1,3 +1,8 @@
+/* Supposed to be coming from webpack entry, but we explicitly import
+   here so karma-webpack will pick it up.
+ */
+@import '@folio/stripes-components/lib/global';
```

Comment:
I'm curious... I think maybe in production this would duplicate the styles? They're included once by webpack, but if they're imported here, I don't think postcss will scrub them out properly (because it doesn't know they're being added by webpack).

I agree with @ttamj that this could probably go in `/tests/harness.js`. We have a css loader, so `import '@folio/stripes-components/lib/global.css'` _should_ work.

Reply:
I assume we could probably do the same with `import 'typeface-source-sans-pro'`